docs/release-notes/8.9.asciidoc

```diff
@@ -0,0 +1,58 @@
+// THIS ALSO MEANS IF YOU USE LINKS HERE, THEY SHOULD BE FULL URLS WITH NO ATTRIBUTES
+
+:pull: https://github.com/elastic/kibana/pull/
+
+There are no breaking changes in 8.9.0.
+
+//end::breaking-changes[]
+
+[discrete]
+[[deprecations-8.9.0]]
+==== Deprecations
+* Removes the option to use the legacy navigation menu ({pull}158094[#158094]).
+
+[discrete]
+[[features-8.9.0]]
+==== New Features
+* Allows you to install the Cloud Security Posture Management (CSPM) integration via CloudFormation ({pull}159994[#159994]).
+* Creates a new dashboard, Cloud Native Vulnerability Management, that provides an overview of vulnerabilities on your cloud hosts ({pull}159699[#159699]).
+* Allows you to group vulnerabilities by resource (host) on the Vulnerabilities Findings page, and creates a Resource flyout that displays detailed vulnerability findings for individual hosts ({pull}159873[#159873], {pull}158987[#158987]).
+* Adds a new custom dashboard, "Detection rule monitoring" ({pull}159875[#159875]).
+* Allows you to anonymize event field values sent to AI Assistant ({pull}159857[#159857]).
+* Adds a *Chat* button that opens AI Assistant to the alert details flyout ({pull}159633[#159633]).
+* Updates AI Assistant to let you create and delete custom system prompts and default conversations ({pull}159365[#159365]).
+* Adds a tour that introduces the EQL missing events feature ({pull}158863[#158863]).
+* Lets you add tags to alerts ({pull}157786[#157786]).
+* Adds a tooltip to snoozed rules showing exactly when alerting will resume ({pull}157407[#157407]).
+
+[discrete]
+[[enhancements-8.9.0]]
+==== Enhancements
+* Adds a *Last response* dropdown menu to the Rules table that allows you to filter rules by the status of their last execution ("Succeeded", "Warning", or "Failed") ({pull}159865[#159865]).
+* Creates a Lens dashboard for monitoring the use of tokens by AI Assistant ({pull}159075[#159075]).
+* Creates a connector for D3 Security ({pull}158569[#158569]).
+* Improves the interface for adding and updating Elastic prebuilt rules ({pull}158450[#158450]).
+* Shows a rule's actions on its details flyout ({pull}158189[#158189]).
+* Allows you to add Lens visualizations to cases from the visualization's *More actions* menu ({pull}154918[#154918]).
+
+[discrete]
+[[bug-fixes-8.9.0]]
+==== Bug fixes
+* Fixes a bug that restricted the number of cloud accounts which could appear on the Cloud Security Posture dashboard to 10 ({pull}157233[#157233]).
+* Fixes a bug that allowed you to save a rule that has an alert filter missing a query ({pull}159690[#159690]).
+* Fixes unexpected filtering behavior on the Alert page. When you select a filter that excludes all alerts, an empty table now appears ({pull}160374[#160374]).
+* Improves input validation for investigation guide queries ({pull}160574[#160574], {pull}160577[#160577]).
+* Improves the handling of special characters in case tags ({pull}159815[#159815]).
```

Comment:
Are these really bugs -- or enhancements?

Reply:
Thanks for the extra pair of eyes on this, @jmikell821! My responses below:
- #160574:  Looks like the PR fixes a UI bug where the **Label** field in the Investigation Guide form incorrectly turns red when the entered value is correct. Do you think we should modify the release notes summary so that it's closer to this description? 
- #159815:  This is a ResponseOps PR that should be doc'd in the Kibana release notes, so I'm removing it.

Reply:
@nastasha-solomon re: #160574 - we can leave it as a bug, but let's start it with `Fixes`. I'm a +1 for revising it to the description you noted.

Reply:
Resolved in [397d33a](https://github.com/elastic/security-docs/pull/3536/commits/397d33a9dfa887ce7bb85fea5ef5df2c9a04b1c1).